package.json

```diff
@@ -29,5 +23,17 @@
   "jspm": {
     "main": "lib/VSS.SDK",
     "format": "global"
+  },
+  "types": "./typings/index.d.ts",
+  "dependencies": {
+    "@types/jquery": "^2.0.34",
+    "@types/jqueryui": "^1.11.31",
+    "@types/knockout": "^3.4.38",
+    "@types/q": "0.0.32",
+    "@types/react": "^0.14.54",
+    "@types/requirejs": "^2.1.28"
+  },
```

Comment:
What about using the `main` attribute of package.json to point to VSS.SDK.js? Someone could, theoretically, pull in the VSS SDK via their own module loader, right? From NPM docs:

> The main field is a module ID that is the primary entry point to your program. That is, if your package is named foo, and a user installs it, and then does require("foo"), then your main module's exports object will be returned.
> 
> This should be a module ID relative to the root of your package folder.
> 
> For most modules, it makes the most sense to have a main script and often not much else.